modules/terminal.nix

```
@@ -0,0 +1,52 @@
+# Copyright (c) 2019-2022, see AUTHORS. Licensed under MIT License, see LICENSE.
+
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.terminal;
+in
+
+{
+
+  ###### interface
+
+  options = {
+
+    terminal.font = mkOption {
+      default = null;
+      type = types.nullOr types.path;
+      example = "${pkgs.terminus_font_ttf}/share/fonts/truetype/TerminusTTF.ttf";
```

That will render a more usable example.
```suggestion
      example = literalExpression ''"''${pkgs.terminus_font_ttf}/share/fonts/truetype/TerminusTTF.ttf"'';
```